zeebe-cluster/templates/_helpers.tpl

```diff
@@ -47,3 +47,16 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- define "zeebe-cluster.version" -}}
 {{- printf "%s:%s" .Values.image.repository .Values.image.tag -}}
 {{- end -}}
+
+{{/*
+Compute the maximum number of unavailable replicas for the broker's PodDisruptionBudget.
+If podDisruptionBudget.maxUnavailable is explicitly set, always use that.
+Otherwise this is calculated as floor(replicationFactor / 2).
+*/}}
+{{- define "broker.podDisruptionBudget.maxUnavailable" -}}
+{{- if (not (kindIs "invalid" .Values.podDisruptionBudget.maxUnavailable)) -}}
```

Comment:
This is the best way I could get this to work when `podDisruptionBudget.maxUnavailable` is explicitly set to 0, since `0` is evaluated as false. See https://github.com/Masterminds/sprig/issues/53 for more discussion.

Reply:
I am ok with this, @falko Are you happy with those calculations for now? @npepinpe ?

Reply:
Sounds good.